Automattic/jetpack
# AI Featured Image: Introduce Carrousel

Closes #36824 

## Proposed changes:

* Add new `Carrousel` component.
* Use `Carrousel` in `Featured Image`.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Create a post.
* Generate Image using `AI Assistant`.
* You should be able to generate new images and see all them, and select the choose one.
* Scenarios to test:
  * An error happened while generating a second/third/... image (you can for it in your sandbox).
  * No enough requests for the next image.

### Demo

_Regular Path_
<video src="https://github.com/Automattic/jetpack/assets/1663717/5081e554-394b-42f0-970e-d8d34a346e13" />

_Loading_
![CleanShot 2024-04-10 at 14 59 36@2x](https://github.com/Automattic/jetpack/assets/1663717/477a41a8-40f0-4d9f-b66c-5a08bdd94895)

_Error_
![CleanShot 2024-04-10 at 15 00 25@2x](https://github.com/Automattic/jetpack/assets/1663717/6d5f9d00-a204-488d-955a-978f9816603b)

_Upgrade on middle_
<video src="https://github.com/Automattic/jetpack/assets/1663717/a7996174-27b4-4736-b327-461797c2feeb" />
